tbx/blog/models.py

```diff
@@ -80,6 +80,9 @@ def serve(self, request):
                 "url": blog_post.url,
                 "author": blog_post.first_author,
                 "date": blog_post.date,
+                "read_time": blog_post.read_time,
```

Comment:
@engineervix are these changes ok to the model?

Reply:
@helenb that looks good to me, this is the same as what was done on the `BlogPage`

Reply:
ah thanks - I thought I had rebased that so not sure why this is showing as different. I'll rebase again to make sure.

Reply:
It is up to date - I think this is showing just because I changed it as well before I rebased once your branch was merged.